library/std/src/sys/pal/unix/weak.rs

```diff
@@ -168,14 +168,7 @@ pub(crate) macro syscall {
             if let Some(fun) = $name.get() {
                 fun($($arg_name),*)
             } else {
-                // This looks like a hack, but concat_idents only accepts idents
-                // (not paths).
-                use libc::*;
```

Comment:
no this absolutely was a brutal hack.

Reply:
a beautiful example of why it eas terrible